docs/getting-started.md

```
@@ -0,0 +1,200 @@
+## 3. Run the app
+
+From the **root** of the repository:
+
+```bash
+yarn dev
+```
+
+This runs `turbo run dev`, which builds the required packages and starts the
+Meteor app in `apps/meteor`. Once you see the server listening, open:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)